src/ssh-generator/20-systemd-ssh-proxy.conf.in

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Make sure unix/* and vsock/* can be used to connect to AF_UNIX and AF_VSOCK paths
+#
+Host unix/* vsock/*
```

Comment:
Perhaps we can use something that looks more URI-like, for example `unix:///<path>` and `vsock://<cid>`.

This rule matches them for ssh:
`Match originalhost="unix:///*,vsock://"`

Reply:
No, this is not an option. The thing is that sftp/scp and similar things all use `:` already, to separate host from path. But we want to allow sftp/scp to also *just work* with these host specifications, hence we need a separator that ssh/sftp/scp do not consider magic here, which rules out at least `@`, and `:`, but leaves `/`. 

Moreover, ssh actually supports an URL-based syntax itself, see man page, it looks like this:

`ssh://[user@]hostname[:port]`

now if we'd insert `vsock://` into that, it would look like this:

`ssh://lennart@vsock://4711:38`

and yuck, i don#t even know how to read this.

All this goes away, if we use the `unix/` or `vsock/` prefix:

`scp *.txt vsock/4711:subdir/`
`ssh ssh://lennart@vsock/4711:45`